src/fork.rs

```diff
@@ -182,15 +181,15 @@ impl<S: ForkSource> ForkStorage<S> {
 
 impl<S: std::fmt::Debug + ForkSource> ReadStorage for ForkStorage<S> {
     fn is_write_initial(&mut self, key: &StorageKey) -> bool {
-        self.is_write_initial_internal(key)
+        self.is_write_initial_internal(key).unwrap()
```

Comment:
Should these methods more gracefully handle `None`? We've tried to get rid/avoid `.unwrap()` as much as possible.

Reply:
Well, they should, but how? They're implementing a trait which doesn't consider the possibility of failure, and which must be implemented to use `ForkStorage` inside EraVM...

Reply:
Discussed offline and agreed this is best to match existing EraVM logic/expectations